src/components/LoginPage/LoginPageComponent.js

```diff
@@ -1,67 +1,96 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import slide1 from 'components/LoginPage/images/slide-1.png';
-import slide2 from 'components/LoginPage/images/slide-2.png';
-import slide3 from 'components/LoginPage/images/slide-3.png';
-import slide4 from 'components/LoginPage/images/slide-4.png';
+import styled from 'styled-components';
+import Slide1 from 'components/LoginPage/Slide1';
+import Slide2 from 'components/LoginPage/Slide2';
+import Slide3 from 'components/LoginPage/Slide3';
+import Slide4 from 'components/LoginPage/Slide4';
+import Slide5 from 'components/LoginPage/Slide5';
+import iconGoogle from 'components/LoginPage/images/icon-google.svg';
+import COLORS from 'components/LoginPage/colors';
+import { ASPECT_RATIO} from 'components/LoginPage/slideComponents';
+import { FONT_SIZE } from 'components/LoginPage/slideComponents';
+
+const Main = styled.main`
+  width: 100%;
+  background-color: ${COLORS.rainfull};
+  font-family: Roboto;
+`;
+
+const Article = styled.article`
+  width: 100%;
+  max-width: calc(100vh * ${ASPECT_RATIO});
+  margin: auto;
+`;
+
+const H3 = styled.h3`
+  padding-bottom: 7%;
+  font-size: ${FONT_SIZE.H3};
+  text-align: center;
+  color: ${COLORS.white};
+`;
+
+const Button = styled.button`
```

Comment:
This button has styles specifically for the google login button.
So let's call it `GoogleLoginButton`.

Reply:
This button also requires using `   <img src={iconGoogle} />` inside.
So ideally it it would be a `GoogleLoginButton` component which renders the image and the text.

Reply:
Sorry, I don’t understand what exactly needs to be done (your second comment)
Some changes are in [d9f4bc6](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/d9f4bc67bc0ba7a51b821e89ce7bdb3b5e9c7952)